README.md

```diff
@@ -150,6 +150,10 @@ open build/html/index.html
 
 See [here](https://github.com/saleml/torchgfn/tree/master/tutorials/ENV.md)
 
+### Defining a GFlowNet
```

Comment:
Thank you for adding GFLOWNET.md. 
Small nitpick, the word GFlowNet can be understood in different ways. Some people would understand "Defining a GFlowNet" as "creating a model then using an existing loss". I believe it makes more sense to add the section this a bit later in the README, and name it some thing like "Extending GFlowNets" or "Creating a new GFlowNet loss", or something like that. Same thing for tutorials/README.md ?

What do you think ?